core/appeal/service.go

```diff
@@ -491,6 +496,402 @@ func validateAppealOnBehalf(a *domain.Appeal, policy *domain.Policy) error {
 	return nil
 }
 
+// Patch record
+func (s *Service) Patch(ctx context.Context, appeal *domain.Appeal) error {
+	existingAppeal, err := s.GetByID(ctx, appeal.ID)
+	if err != nil {
+		return fmt.Errorf("error getting existing appeal: %w", err)
+	}
+
+	if existingAppeal.Status != domain.AppealStatusPending {
+		return ErrAppealStatusInvalid
```

Comment:
```
ErrAppealStatusInvalid = errors.New("invalid status")
```
```suggestion
		return fmt.Errorf("%w: unable to edit appeal in status: %q", ErrAppealStatusInvalid, existingAppeal.Status)
```
this would be a more helpful error message for user

Reply:
small, but let's update this as well 🙏
```
ErrAppealStatusInvalid = errors.New("invalid status")
```